src/styles/_filter.scss

```diff
@@ -0,0 +1,37 @@
+.filter {
+    align-items: center;
+    background-color: $white;
+    border-radius: $border-radius;
+    display: flex;
+    flex-wrap: wrap;
+    font-family: $font-jost;
+    height: 165px;
+    justify-content: center;
+    width: 250px;
+    
+    div {
+        display: flex;
+        flex-wrap: wrap;
+        height: 85%;
+        width: 85%;
+    }
+
+    button {
+        background-color: $mid-gray;
+        border-radius: $border-radius;
+        border: none;
+        color: $blurple;
+        font-size: 14px;
+        font-weight: 600;
+        height: 30px;
+        line-height: 19px;
+        margin-right: 10px;
+        padding-left: 15px;
+        padding-right: 15px;
+    }
+
+    button:hover {
```

Comment:
Suggestion to move this into the `button` style

Reply:
Moved